aten/src/ATen/native/quantized/cpu/qnnpack/src/q8gemm/8x8-aarch64-neon.S

```diff
@@ -99,9 +105,12 @@ BEGIN_FUNCTION pytorch_q8gemm_ukernel_8x8__aarch64_neon
     # v23 := vacc7x4567
     MOV v23.16b, v9.16b
 
+    # Add offset to the base pointer
+    ADD x17, x17, x16
     // Load requantization_scale
-    // - v26 = requantization_scale 
-    LD1R {v26.4s}, [x8], 4
+    // - v26 = requantization_scale channels 0-3
+    // - v27 = requantization_scale channels 4-7
+    LD1 {v26.4s}, [x17], 16
 
```

Comment:
At a glance it seems to me that you can save a couple of instructions in the block above if interested.  It basically boils down to modifying the pointer arithmetic to take advantage of the free offsetting in the loads.

Reply:
Yes so pointer arithmetic can save lsl by folding it in. LD1 I dont think since it has only no offset and post-index variants.

Reply:
I am gonna make these changes in perf opt PR.